src/app/account-settings-page/account-settings-page.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="block-user-table-container mat-elevation-z8">
+  <div class="block-user-table-header">
+    <h3>Blocked users</h3>
```

Comment:
You should use a `mat-title` here instead of an `h3` so that the font face and font size is the same as the headers for other pages.

Reply:
you probably meant mat-card-title